src/EditorFeatures/Core/Implementation/ReferenceHighlighting/AbstractDocumentHighlightsService.cs

```diff
@@ -113,8 +114,13 @@ private bool ShouldConsiderSymbol(ISymbol symbol)
         private Task<IEnumerable<Location>> GetAdditionalReferencesAsync(
             Document document, ISymbol symbol, CancellationToken cancellationToken)
         {
-            return document.Project.LanguageServices.GetService<IReferenceHighlightingAdditionalReferenceProvider>()
-                .GetAdditionalReferencesAsync(document, symbol, cancellationToken);
+            var additionalReferenceProvider = document.Project.LanguageServices.GetService<IReferenceHighlightingAdditionalReferenceProvider>();
```

Comment:
I was testing the scenario when a document contained both C# blocks and JS script blocks. The call to:
document.Project.LanguageServices.GetService<IReferenceHighlightingAdditionalReferenceProvider>() .GetAdditionalReferencesAsync(document, symbol, cancellationToken) was throwing a null reference exception.  Added the null check to support this scenario.